### PR TITLE
new annotatations

### PR DESCRIPTION
**Pull request by redxaxder**

### CODE REVIEW

**Review by nc6 — APPROVED** [review]
Went through this with Alex today, looks good to me. There will be a couple of follow-up issues.

File: stack.yaml [hunk 1]
```diff
@@ -15,13 +15,13 @@ extra-deps:
...
   - git: https://github.com/input-output-hk/cardano-prelude
-    commit: b9bf62f2bab90539809bee13620fe7d29b35928d
+    commit: e5d847e8ba5a3ce87c5f8010e50c783f700343ec
     subdirs:
       - .
       - test
 
   - git: https://github.com/input-output-hk/cardano-base
-    commit: ccbc085f1ebdde706feb7955fdebb33260dc5e32
+    commit: dff21d7a46d5bbe257f0ce2d4f777d69dfb9f9b7
```
Comment (intricate) [inline stack.yaml]: Dependency referencing a commit that isn't on `master`.

File: stack.yaml [hunk 2]
```diff
@@ -15,13 +15,13 @@ extra-deps:
   - generic-monoid-0.1.0.0
 
   - git: https://github.com/input-output-hk/cardano-prelude
-    commit: b9bf62f2bab90539809bee13620fe7d29b35928d
+    commit: e5d847e8ba5a3ce87c5f8010e50c783f700343ec
```
Comment (intricate) [inline stack.yaml]: Dependency referencing a commit that isn't on `master`.

**Review by intricate — DISMISSED** [review]
Haven't reviewed all of the content yet, but just making sure that this can't be merged until the required changes for `cardano-base` and `cardano-prelude` are merged to `master`.

**Comment by nc6** [comment]
bors retry

**Pull request merged**